tests/console/yast2_nis.pm

```diff
@@ -62,9 +62,11 @@ sub run() {
     send_key 'alt-y';
     wait_screen_change { type_string "-c" };                             # only checks if the config file has syntax errors and exits
     wait_screen_change { send_key 'alt-o' };
-    send_key_until_needlematch('nfs-client-configuration', 'alt-s', 2, 5);    # enter NFS configuration, sometimes fails, retry
-    send_key 'alt-a';                                                         # add nfs settings
-    assert_screen 'nfs-server-hostname';                                      # check that type string is sucessful
+    wait_screen_change { send_key 'alt-s' };                             # enter NFS configuration,sometimes fails, retry
+    send_key 'alt-s' if check_screen('nis-client');                      # Cannot use send_key_until_needlematch, see poo#48647
```

Comment:
Fails: http://amazing.suse.cz/tests/5032#step/yast2_nis/17. The only option that I found always working (in 160 Vrs) is to use sleep(4).